source/MaterialXGenShader/Shader.h

```diff
@@ -48,19 +48,39 @@ class Shader
         DOWN
     };
 
+    struct Variable;
+    using VariablePtr = std::shared_ptr<Variable>;
```

Comment:
Nitpick: Can be merged to `using VariablePtr = std::shared_ptr<struct Variable>;`